src/main/java/com/scalar/db/storage/multistorage/MultiStorageConfig.java

```diff
@@ -88,17 +91,34 @@ private void loadTableStorageMapping() {
     Builder<String, String> builder = ImmutableMap.builder();
 
     String tableMapping = props.getProperty(TABLE_MAPPING);
-    for (String tableAndStorage : tableMapping.split(",")) {
-      String[] s = tableAndStorage.split(":");
-      String table = s[0];
-      String storage = s[1];
-
-      checkIfStorageExists(storage);
-      builder.put(table, storage);
+    if (tableMapping != null) {
```

Comment:
It is a minor style thing, but I feel the following is slightly better since less indenting.
```
if (tableMapping == null) {
  return null;
}
for (String tableAndStorage : tableMapping.split(",")) {
  // ...
}
```